st2client/st2client/commands/action_alias.py

```diff
@@ -127,3 +132,89 @@ def run_and_print(self, args, **kwargs):
         print("Matching Action-alias: '%s'" % execution.actionalias['ref'])
         print("To get the results, execute:\n st2 execution get %s" %
               (execution.execution['id']))
+
+
+class ActionAliasTestCommand(ActionRunCommandMixin, resource.ResourceCommand):
+    display_attributes = ['name']
+
+    def __init__(self, resource, *args, **kwargs):
+        super(ActionAliasTestCommand, self).__init__(
+            resource, 'test',
+            ('Execute the command text by finding a matching %s and format the result.' %
+             resource.get_display_name().lower()), *args, **kwargs)
+
+        self.parser.add_argument('command_text',
+                                 metavar='command',
+                                 help=('Execute the command text by finding a matching %s.' %
+                                       resource.get_display_name().lower()))
+        self.parser.add_argument('-u', '--user', type=str, default=None,
+                                 help='User under which to run the action (admins only).')
+
+        self._add_common_options()
+        self.parser.add_argument('-a', '--async',
+                                 action='store_true', dest='action_async',
+                                 help='Do not wait for action to finish.')
+
+    @resource.add_auth_token_to_kwargs_from_cli
+    def run(self, args, **kwargs):
+        payload = core.Resource()
+        payload.command = args.command_text
+        payload.user = args.user or ""
+        payload.source_channel = 'cli'
+
+        alias_execution_mgr = self.app.client.managers['ActionAliasExecution']
+        execution = alias_execution_mgr.match_and_execute(payload)
+        return execution
+
+    def run_and_print(self, args, **kwargs):
+        # 1. Trigger the execution via alias
+        print("Triggering execution via action alias")
+        print("")
+
+        # NOTE: This will return an error and abort if command matches no aliases so no additional
+        # checks are needed
+        result = self.run(args, **kwargs)
+        execution = Execution.deserialize(result.execution)
+
+        # 2. Wait for it to complete
+        print("Execution (%s) has been started, waiting for it to finish..." % (execution.id))
```

Comment:
Since we're only on Python 3 now, using f-strings would be great!

```suggestion
        print(f"Execution ({execution.id}) has been started, waiting for it to finish...")
```